backend/prisma/migrations/20240715220240_init_words/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- CreateTable
+CREATE TABLE "Words" (
+    "id" SERIAL NOT NULL,
+    "usableWords" TEXT[],
+    "deletedWords" TEXT[],
+
+    CONSTRAINT "Words_pkey" PRIMARY KEY ("id")
+);
```

Comment:
this is likely not the right structure here. Ideally we would instead have something like this:

```
CREATE TABLE "Words" (
 "id" SERIAL NOT NULL,
 "word" TEXT,
 "category" TEXT, // either usable or deleted
)
```

Then each word added gets its own row.

One of the main strengths of databases is efficient querying and filtering over entries. If you store everything in an array like this we lose these features. In general, having arrays directly containing the data you want to store inside of a database is usually not what you want to do.

Reply:
related note: I would prefer to use an Enum https://www.postgresql.org/docs/current/datatype-enum.html or a `smallint` for the "category", since indices on these types perform better than indices on TEXT columns.

Reply:
unrelated note: You probably want to have some notion of a `game_id` for these tables too, since your list of "usable" words might be different from game to game.